tools/write-filesizes.js

```diff
@@ -0,0 +1,31 @@
+const mkdirp = require('mkdirp');
+const { join } = require('path');
+const yargs = require('yargs').argv;
+const { getFileSizes } = require('./compareFilesize');
+const log = require('./gulptasks/lib/log');
+
+const files = ['highcharts.src.js'];
```

Comment:
This is the source files with comments and all. We want to inspect the compiled file, `highcharts.js`.

Reply:
Inspecting the `getFileSizes` function, I realize this is built in..